blocks/api/serializer.js

```diff
@@ -50,20 +50,28 @@ export function getSaveContent( blockType, attributes ) {
 	}
 
 	// Adding a generic classname
-	const addClassnameToElement = ( element ) => {
-		if ( ! element || ! isObject( element ) || ! className ) {
+	const addGenericAttributes = ( element ) => {
```

Comment:
Not sure if @jasmussen has other thoughts on how to separate advanced features and make them less pronounced. It may be easier to collapse in place instead of at the footer.